src/screens/ContactScreen/index.js

```diff
@@ -166,26 +230,19 @@ const ContactScreen = ({navigation}) => {
   const onRefresh = React.useCallback(async () => {
     setRefreshing(true);
     try {
-      const res = await userPopulate();
-      setUsers(res);
-      setCacheUser(res);
+      getDiscoveryUser();
       setRefreshing(false);
     } catch (error) {
       setRefreshing(false);
     }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getDiscoveryUser` function is now called inside the `onRefresh` callback instead of `userPopulate`. Ensure that this new function correctly fetches and sets the user data.
```diff
- const res = await userPopulate();
- setUsers(res);
- setCacheUser(res);
+ getDiscoveryUser();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->